draft-scenarios/clash/king_of_the_hill_v2.tex

```diff
@@ -0,0 +1,130 @@
+% !TeX spellcheck = en_US
+\addscenariosection{1}{Clash Scenario}{King of the Hill v2}{\images/kingofthehill.png}
+
+\begin{multicols}{2}
+
+\textbf{Author:} KABLAMIO \& Cantila
+
+\textbf{Source:} \href{https://discord.com/channels/1275490124301467799/1284428969369665631}{KABLAMIO's Discord}
+
+\textit{All eyes turn to the Hill — where legends rise and empires fall. Fight, claim it, and prove you’re worthy to rule.}
+
+\subsection*{\MakeUppercase{Scenario Length}}
+This Scenario plays out over 6 Rounds.
+
+\subsection*{\MakeUppercase{Player Setup}}
+\textbf{Player Count:} 2 -- 3
+
+\textbf{Starting Resources:} 22 \svg{gold}, 5 \svg{building_materials}
+
+\textbf{Starting Income:} 15 \svg{gold}, 2 \svg{building_materials}, 1 \svg{valuables}
+
+\textbf{Starting Units:}
+\begin{itemize}
+  \item A Few of cheapest \silver\ Units
+\end{itemize}
+
+\textbf{Town Buildings:}
+\begin{itemize}
+  \item \bronze\ Dwelling
+\end{itemize}
+
+\subsection*{\MakeUppercase{Map Setup}}
+Take the following Map Tiles and arrange them as shown in the Scenario map layout ($P$ stands for the number of players):
+
+\begin{itemize}
+  \item P × Starting (I) Map Tile
+  \item P × Far (II-III) Map Tiles
+  \item P × Near (IV-V) Map Tiles
+  \item Center Map Tile (C1 or C3)
+  \item Custom Center Map Tile
```

Comment:
I wonder if it needs any additional explanation, even something as simple as "see map layout", as there are no other scenarios with custom tiles, at least in this book.